packages/app_database/analysis_options.yaml

```
@@ -0,0 +1,9 @@
+include: package:sizzle_lints/lints.yaml
+
+analyzer:
+  exclude: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)